### DISCUSSION TITLE
v4.2.0

### DISCUSSION THREAD

**Discussion by devlead** [cake-build/cake#4376] [Announcements]
As part of this release we had [1 issue](https://github.com/cake-build/cake/milestone/92?closed=1) closed.

__Bug__

- [__#4374__](https://github.com/cake-build/cake/issues/4374) Argument 'foo' was not set" after update to 4.1 in Cake Frosting

<hr /><em>This discussion was created from the release <a href='https://github.com/cake-build/cake/releases/tag/v4.2.0'>v4.2.0</a>.</em>

**Comment by Yaslongi** [comment]
hello im new
